include/exadg/operators/inverse_mass_operator.h

```diff
@@ -60,14 +62,30 @@ class InverseMassOperator
     this->matrix_free = &matrix_free_in;
     dof_index         = dof_index_in;
     quad_index        = quad_index_in;
+
+    explicit_matrix_free_inverse_mass_available = matrix_free->get_dof_handler(dof_index)
```

Comment:
I think this is not enough? We only have the explicit matrix-free inverse mass for DG elements (L2 conforming), so that the mass matrix is block-diagonal.

Reply:
Yes, see the discussion above https://github.com/exadg/exadg/pull/380#discussion_r1134306730